topics/sentinel.md

```diff
@@ -1239,6 +1239,21 @@ When in TILT mode the Sentinel will continue to monitor everything, but:
 * It starts to reply negatively to `SENTINEL is-master-down-by-addr` requests as the ability to detect a failure is no longer trusted.
 
 If everything appears to be normal for 30 second, the TILT mode is exited.
+        
+In the Sentinel Tilt Mode, if we send INFO command, we could get the following response:
+    127.0.0.1:26379>info
+    //*** Other information from Sentinel server
+
+    # Sentinel
+    sentinel_masters:1
+    sentinel_tilt:0
+    sentinel_tilt_since_seconds:-1
+    sentinel_running_scripts:0
+    sentinel_scripts_queue_length:0
+    sentinel_simulate_failure_flags:0
+    master0:name=mymaster,status=ok,address=127.0.0.1:6379,slaves=0,sentinels=1
+        
+The Field "sentinel_tilt_since_seconds" indicates how many millisecond Sentinel already is in the Tilt mode. If it is not in the Tilt mode, the value will be -1
```

Comment:
```suggestion
The field `sentinel_tilt_since_seconds` indicates how many seconds Sentinel already is in the TILK mode.
If it is not in the TILT mode, the value will be -1.
```

`sentinel_tilt_since_seconds` is seconds, not milliseconds, right?

Reply:
Yes, it is second, I will update it